src/internal.rs

```diff
@@ -505,27 +507,49 @@ pub struct Rav1dFrameContext_frame_thread {
 #[derive(Default)]
 pub(crate) struct TxLpfRightEdge {
     /// `.len() = h * 2`
-    inner: Vec<u8>,
+    inner: DisjointMut<Vec<u8>>,
 }
 
 impl TxLpfRightEdge {
     #[allow(dead_code)]
-    pub const fn new() -> Self {
-        Self { inner: Vec::new() }
+    pub fn new() -> Self {
+        Self {
+            inner: DisjointMut::default(),
+        }
```

Comment:
The `fn new` here was specifically a `const fn new`, so if we're not gonna make it `const`, there's no point in having it, as we already `#[derive(Default)]`.